gk/main.c

```diff
@@ -1643,8 +1643,12 @@ process_pkts_back(uint16_t port_back, uint16_t port_front,
 
 		fib = look_up_fib(&gk_conf->lpm_tbl, &packet.flow);
 
-		 /* No entry for the destination, drop the packet. */
-		if (fib == NULL) {
+		 /*
+		  * No entry for the destination or the entry instructs
+		  * to forward the packet to the neighbor in the back network,
+		  * drop the packet.
+		  */
+		if (fib == NULL || fib->action == GK_FWD_NEIGHBOR_BACK_NET) {
 			if (packet.flow.proto == ETHER_TYPE_IPv4)
 				add_pkt_acl(acl4, pkt);
 			else if (likely(packet.flow.proto ==
```

Comment:
I can't comment at the `switch..case` below, but could add `case GK_DROP:`? A routing daemon may add this FIB type.